calliope/backend/run.py

```diff
@@ -500,14 +515,15 @@ def run_operate(model_data, timings, backend, build_only):
                 comment="Backend: iteration {}: sending model to solver".format(i + 1),
             )
             # After iteration 1, warmstart = True, which should speed up the process
-            # Note: Warmstart isn't possible with GLPK (dealt with later on)
+            # Note: Warmstart isn't possible with GLPK (dealt with here to avoid the warning being duplicated)
+
             _results = backend.solve_model(
                 backend_model,
                 solver=solver,
                 solver_io=solver_io,
                 solver_options=solver_options,
                 save_logs=save_logs,
-                warmstart=warmstart,
+                warmstart=warmstart * allow_warmstart,
```

Comment:
Is this actually needed? i.e. will it error on a solver that cannot warmstart, if warmstart=True?

Reply:
Yes, it will, annoyingly.